### ISSUE TITLE
Wrong `slash` and `backslash` in the `sym` module

### ISSUE THREAD

**Issue by johanvx** [typst/typst#191]
The symbol `slash` is currently [Solidus] `\u{002F}` (which is in the "Basic Latin" Block), while it should be the mathematical operator [Division Slash] `\u{2215}` (which is in the "Mathematical Operators" Block), as `slash.double` ([Double Solidus Operator] `\u{2AFD}`) and `slash.triple` ([Triple Solidus Binary Relation]) are also mathematical operators.

Similarly, since `backslash.not` is a math operator/symbol ([Reverse Solidus with Horizontal Stroke] `\u{29F7}`), `backslash` should be [Reverse Solidus Operator] `\u{29F5}` instead of [Reverse Solidus] `\u{005C}`.

---

~By the way, [Set Minus] `\u{2216}` and Reverse Solidus Operator are different, so it would be great to add a `setminus` symbol.~ Ah I see, it's `without`.

[Solidus]: https://www.compart.com/en/unicode/U+002F
[Division Slash]: https://www.compart.com/en/unicode/U+2215
[Double Solidus Operator]: https://www.compart.com/en/unicode/U+2AFD
[Triple Solidus Binary Relation]: https://www.compart.com/en/unicode/U+2AFB
[Reverse Solidus with Horizontal Stroke]: https://www.compart.com/en/unicode/U+29F7
[Reverse Solidus Operator]: https://www.compart.com/en/unicode/U+29F5
[Reverse Solidus]: https://www.compart.com/en/unicode/U+005C
[Set Minus]: https://www.compart.com/en/unicode/U+2216

**Comment by laurmaedje** [comment]
I think this was intentional because the mathematical slash looks really bad in the math font. But not 100% sure.

**Comment by johanvx** [comment]
I agree... I've also noticed that the LaTeX output for `/` in $2 / 3$ is also Solidus, so I consider this to be an accepted convention. Maybe it's just because Solidus is more beautiful 😂 . Let's keep Solidus.

Therefore, since `slash` is in the "Basic Latin" Block, `backslash`, as the mirror of `slash`, should be in the same block, which means that we also keep currently used Reverse Solidus.

In conclusion, nothing needs to be changed! I think this issue can be closed now if it is confirmed that we won't change `slash` and `backslash` (at least for now).
